### PR TITLE
Adding NamedNodeMap to global export

### PR DESCRIPTION
**Pull request by mikemadest**
The reason for this PR is this error when running Axe tests in a React app, 
using Jest with a linkedom environment:

image

And this is the axe test involved:
```javascript
    function getNodeAttributes(node) {
      if (node.attributes instanceof window.NamedNodeMap) {
        return node.attributes;
      }
      return node.cloneNode(false).attributes;
    }
```

What happen is that `NamedNodeMap`, despite being in linkedom interface, wasn't available in `window`, which is what this PR intend to change.

### CODE REVIEW

File: test/interface/named-node-map.js [hunk 1]
```diff
@@ -2,10 +2,10 @@ const assert = require('../assert.js').for('NamedNodeMap');
 
 const {parseHTML} = global[Symbol.for('linkedom')];
 
-const {document} = parseHTML('<html><div tmp>abc</div></html>');
+const {document, NamedNodeMap} = parseHTML('<html><div tmp>abc</div></html>');
 
 let node = document.documentElement.firstElementChild;
-
+assert(typeof NamedNodeMap !== 'undefined', true, 'NamedNodeMap undefined in global export');
```
Comment (mikemadest) [inline test/interface/named-node-map.js:8]: My intention was to add a unit test related to the change here. I couldn't find similar checks for other exported objects but felt bad not adding something...

File: esm/interface/document.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ import {InputEvent} from './input-event.js';
 import {ImageClass} from './image.js';
 import {MutationObserverClass} from './mutation-observer.js';
 import {NodeList} from './node-list.js';
+import {NamedNodeMap} from './named-node-map.js';
```
Comment (WebReflection) [inline esm/interface/document.js:33]: Do you mind moving this above the previous line? Just to preserve the alphabetical order, thanks!

**Comment by WebReflection** [comment]
Thanks, beside one line to move, this looks good to me 👍

File: esm/interface/document.js (hunk 2 above)
Comment (mikemadest) [reply esm/interface/document.js:33]: Of course, thanks for your feedback 👍

Comment (mikemadest) [reply esm/interface/document.js:32]: Done

**Pull request merged**